main.cpp

```diff
@@ -0,0 +1,124 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+class Country{
+	public:
+	char name[30];
+	int xl;
+	int xr;
+	int yl;
+	int yr;
+	int cnt;
+	int day;
+} C[25];
+
+class City {
```

Comment:
Better use struct if you don't have any private methods

Reply:
done